examples/dft/compile_time_dispatching/CMakeLists.txt

```diff
@@ -30,19 +30,17 @@ foreach(dft_ct_source ${DFT_CT_SOURCES})
   add_executable(${EXAMPLE_NAME} ${dft_ct_source}.cpp)
   target_include_directories(${EXAMPLE_NAME}
       PUBLIC ${PROJECT_SOURCE_DIR}/examples/include
-      PUBLIC ${PROJECT_SOURCE_DIR}/include
```

Comment:
Any specific reasons for removing `include`  from this target?

Reply:
Yes.

If the backend target (eg. `onemkl_dft_cufft`) is configured correctly, then linking against the backend target using `target_link_libraries` should bring in this include directory anyway. The fact that it doesn't do this is the motivation for this PR - otherwise the gymnastics [like here](https://gitlab.com/gromacs/gromacs/-/blob/main/src/gromacs/CMakeLists.txt?ref_type=heads#L252) are required.

Reply:
Thanks for details.